src/client/datascience/jupyter/kernels/cellExecution.ts

```diff
@@ -443,23 +448,35 @@ export class CellExecution {
 
         try {
             if (jupyterLab.KernelMessage.isExecuteResultMsg(msg)) {
+                traceInfoIf(!!process.env.VSC_JUPYTER_LOG_KERNEL_OUTPUT, 'KernelMessage = ExecuteResult');
                 await this.handleExecuteResult(msg as KernelMessage.IExecuteResultMsg, clearState);
             } else if (jupyterLab.KernelMessage.isExecuteInputMsg(msg)) {
                 await this.handleExecuteInput(msg as KernelMessage.IExecuteInputMsg, clearState, loggers);
             } else if (jupyterLab.KernelMessage.isStatusMsg(msg)) {
+                traceInfoIf(!!process.env.VSC_JUPYTER_LOG_KERNEL_OUTPUT, 'KernelMessage = StatusMessage');
                 // Status is handled by the result promise. While it is running we are active. Otherwise we're stopped.
                 // So ignore status messages.
                 const statusMsg = msg as KernelMessage.IStatusMsg;
                 this.handleStatusMessage(statusMsg, clearState);
             } else if (jupyterLab.KernelMessage.isStreamMsg(msg)) {
+                traceInfoIf(
```

Comment:
Added to get more logging for a flaky test.
Kinda like .NET tracing.
Not sure if this would introduce a per issue, i do'nt se how.